src/psconvert.c

```diff
@@ -813,6 +816,7 @@ static int usage (struct GMTAPI_CTRL *API, int level) {
 		"Escape any +? modifier inside strings with \\.");
 	if (API->GMT->current.setting.run_mode == GMT_CLASSIC)
 		GMT_Usage (API, 1, "\n-Z Remove input PostScript file(s) after successful conversion.");
+	GMT_Usage (API, 1, "\n-! Modify the input PS file instead of creating a temp EPS. Some options like -Ng will disable it.");
```

Comment:
```suggestion
	GMT_Usage (API, 1, "\n-! Modify the input PS file instead of creating a temp EPS. Some options like -Wg will disable it.");
```
I think you mean `-Wg`, right?

I've tried this branch in PyGMT. Without adding the `-!` option, most tests still work, but with `-Wg`, I see errors like:
```
Error: rt [ERROR]: Could not find the 'PROJ' tag in the PS file. No world file created.
Error: rt [ERROR]: This situation occurs when one of the two next cases is true:
Error: rt [ERROR]: 1) the PS file was created with a pre-GMT v4.5.0 version
Error: rt [ERROR]: 2) the PS file was not created by GMT
```

Is the `-!` option enabled by default? I didn't add `-!`, so it should not affect any existing tests, right?

Reply:
No, it's `-Ng`. This and other have to disable the `-!` (not possible given the particular option requirements). But we need it better explained and to create a dedicated option to specifically ignore what `-!` does.

Reply:
But the error is real, thanks. Have to see what to do about it.

Reply:
There is no `-Ng`, only `-N+g`.